panels/configspec.ini

```diff
@@ -40,3 +40,7 @@ movie = string(default="Media presentation (like a movie)")
 performance = string(default="Scripted or Improv Live Performance")
 participation = string(default="Audience participation (like a game)")
 other = string(default="Other")
+
+[[panel_vote]]
+accept = string(default="Accept")
+decline = string(default="Reject")
```

Comment:
I made this an enum instead of a boolean since I figured they might want more than just yes/no in the future.